expat/configure.ac

```diff
@@ -222,6 +222,17 @@ AC_DEFINE([XML_DTD], 1,
 AC_DEFINE([XML_DEV_URANDOM], 1,
           [Define to include code reading entropy from `/dev/urandom'.])
 
+AC_ARG_ENABLE([xml-attr-info],
+  [AS_HELP_STRING([--enable-xml-attr-info],
+                 [Enable retrieving the byte offsets for attribute names and values])], ,
+                 [enable_xml_attr_info=yes])
+AS_IF([test "x${enable_xml_attr_info}" = "xyes"],
+      [AC_DEFINE([XML_ATTR_INFO], 1,
+         [Define to allow retrieving the byte offsets for attribute names and values.])]
+      [test "x${enable_xml_attr_info}" = "xno"],
+      [AC_DEFINE([XML_ATTR_INFO], 0,
+         [Define to allow retrieving the byte offsets for attribute names and values.])])
```

Comment:
Is there a way to resolve the code duplication regarding `AC_DEFINE([XML_ATTR_INFO]` and the error text here? `git grep AC_DEFINE` shows me it would be the first time we do/need this duplication.

Reply:
will remove the false case i.e "no", as by default XML_ATTR_INFO is disable.